src/include/quick-lint-js/parse.h

```diff
@@ -154,6 +154,7 @@ class parser {
 
       case token_type::kw_break:
       case token_type::kw_continue:
+      case token_type::kw_debugger:
```

Comment:
Currently, grouping `debugger` with `break` and `continue` makes sense. However, `break` and `continue` can accept a loop label (not implemented in quick-lint-js yet), and I assume `debugger` doesn't accept a loop label. So while the *code* is currently the same, I think the *meanings* are different, so I'd repeat lines 158-160 for `debugger`.